src/test/java/uk/gov/hmcts/reform/slc/logging/AppInsightsTest.java

```diff
@@ -156,11 +162,34 @@ public void should_mark_message_not_handled() {
         );
     }
 
+    @Test
+    public void should_track_letter_not_handled() {
+        Letter letter = new Letter(
+            UUID.randomUUID(),
+            Collections.nCopies(5, new Document(TEMPLATE, Collections.emptyMap())),
+            "type",
+            SERVICE_NAME
+        );
+        insights.trackLetterNotHandled(letter);
+
+        verify(client).trackEvent(
+            AppEvent.LETTER_HANDLED_UNSUCCESSFULLY,
+            ImmutableMap.of(
+                LETTER_ID_KEY, letter.id.toString(),
+                "service", SERVICE_NAME,
+                "type", "type"
+            ),
+            singletonMap("numberOfDocuments", 5.0)
```

Comment:
should no of documents be int rather than double/long?

Reply:
metrics parameter is double

Reply:
🐶